ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/deploymentSingle.test.js

```diff
@@ -0,0 +1,119 @@
+        cy.get(selectors.searchOptionsMenuItem('CVE'));
+        cy.get(selectors.searchOptionsMenuItem('Image'));
+        cy.get(selectors.searchOptionsMenuItem('Component'));
+        cy.get(selectors.searchOptionsMenuItem('Component source'));
+        cy.get(selectors.searchOptionsMenuItem('Deployment')).should('not.exist');
+        cy.get(selectors.searchOptionsMenuItem('Cluster')).should('not.exist');
+        cy.get(selectors.searchOptionsMenuItem('Namespace')).should('not.exist');
+        cy.get(selectors.searchOptionsDropdown).click();
+    });
+
+    it('should navigate between vulnerabilities and resources tabs', () => {
+        visitFirstDeployment();
+
+        // By default, the vulnerabilities tab should be selected
+        cy.get(selectors.vulnerabilitiesTab).should('have.attr', 'aria-selected', 'true');
+        cy.get(selectors.resourcesTab).should('have.attr', 'aria-selected', 'false');
+
+        // Check elements on the Vulnerabilities tab
+        cy.get(selectors.summaryCard('CVEs by severity'));
+        cy.get(selectors.summaryCard('CVEs by status'));
+        // Check table exists with the correct headers
+        cy.get('table thead tr th').contains('CVE');
+        cy.get('table thead tr th').contains('CVE severity');
+        cy.get('table thead tr th').contains('CVE status');
+        cy.get('table thead tr th').contains('Affected components');
+        cy.get('table thead tr th').contains('First discovered');
+
+        // Visit the resources tab
+        cy.get(selectors.resourcesTab).click();
+
+        cy.get(selectors.vulnerabilitiesTab).should('have.attr', 'aria-selected', 'false');
+        cy.get(selectors.resourcesTab).should('have.attr', 'aria-selected', 'true');
+
+        // Check items on the Resources tab
+        // Check table exists with the correct headers
+        cy.get('table thead tr th').contains('Name');
+        cy.get('table thead tr th').contains('Image status');
+        cy.get('table thead tr th').contains('Image OS');
+        cy.get('table thead tr th').contains('Created');
+
+        // Back to the vulnerabilities tab
+        cy.get(selectors.vulnerabilitiesTab).click();
+
+        // Check that pagination is reset across tabs
+        // 1. Manually reload page with a page query param to ensure that the page is not 1
+        // 2. Visit the resources tab
+        // 3. Ensure that the page query param is removed from the URL
+        // 4. Repeat steps 1-3 for the vulnerabilities tab
+        cy.location('href').then((href) => {
+            cy.visit(`${href}?page=2`);
+            cy.get(selectors.resourcesTab).click();
+            cy.location('search').should('not.contain', 'page=');
+        });
+
+        cy.location('href').then((href) => {
+            cy.visit(`${href}?page=2`);
+            cy.get(selectors.vulnerabilitiesTab).click();
+            cy.location('search').should('not.contain', 'page=');
+        });
```

Comment:
Do you think it's worth separating the test into two different ones?
1. Check to see if the tab content shows the correct UI elements (table with specific columns) for the tab
2. Check for the correct pagination state between changing tab views